54.md

```diff
@@ -31,6 +31,71 @@ The content should be Markdown, following the same rules as of [NIP-23](23.md),
 
 One extra functionality is added: **wikilinks**. Unlike normal Markdown links `[]()` that link to webpages, wikilinks `[[]]` link to other articles in the wiki. In this case, the wiki is the entirety of Nostr. Clicking on a wikilink should cause the client to ask relays for events with `d` tags equal to the target of that wikilink.
 
+#### Wikilink Tags
+
+The one to two most prevailing wikilink `d` tag references of an article should be set to the `y` and `z` tags.
+The `y` tag is set to the most prevailing one while `z` is set to the least of the two.
+
+A prevalent wikilink is a reference that is able to categorize the article into a specifc topic. These are usually positioned
+on the top of the article on an introductory paragraph.
+
+A Nostr wiki client may try to auto assign those tags. For example, by finding the first wikilink's paragraph on the article
+then considering the up to first two wikilinks on the current paragraph as the prevailing wikilinks.
+
+Example:
+
+```js
+{
+  "kind": 30818,
+  "tags": [
+    [ "d", "john-doe" ],
+    [ "title", "John Doe" ],
+    [ "y", "internet-celebrity" ],
+    [ "z", "nostr" ],
+  ],
+  "content": "# John Doe\n\n" +
+    "John Doe is an [influencer[internet-celebrity]] in the [Nostr[nostr]] scene. He is known for his [memes[meme]].\n" +
+    "# Early Life\n\n" +
+    "He was born in [Los Angeles[los-angeles]] ...",
+  // other fields
+}
+```
+
+Those tags are handy when searching for topics related to an article.
+They may be used as labels to tell two articles with the same `d` tag apart.
+They are also used when [ranking authors](#author-ranking).
```

Comment:
Having `y` and `z` tags instead of just `t` tags for these use-cases feels overly complicated.

Reply:
Agree. Though used `r` instead of `t`. People tend to over use hashtags and I think the `d` value would end up being a `t` tag too, which is not the intent.